src/Cli/dotnet/CommonOptions.cs

```diff
@@ -281,10 +281,10 @@ internal static CliArgument<T> AddCompletions<T>(this CliArgument<T> argument, F
 
     public enum VerbosityOptions
     {
-        quiet,
-        q,
         minimal,
         m,
+        quiet,
```

Comment:
Can you undo this move? They were previously sorted in order of verbosity, and now they aren't sorted at all.

Reply:
That's a really good point. I was trying to make `minimal` the default option. Let me try to find other ways to make it default. Thanks!